Formula/jenv.rb

```diff
@@ -15,12 +15,24 @@ def install
   end
 
   def caveats
-    <<~EOS
-      To activate jenv, add the following to your #{shell_profile}:
+    if preferred == :fish
+      s = <<~EOS
+        To activate jenv, run the following commands:
 
-        export PATH="$HOME/.jenv/bin:$PATH"
-        eval "$(jenv init -)"
-    EOS
+      EOS
+      s << "  fish_add_path #{bin}\n" if bin.directory?
+      s << "  echo 'status --is-interactive; and source (jenv init -|psub)' >> #{shell_profile}\n"
+      s << "  cp #{libexec}/fish/jenv.fish ~/.config/fish/functions/jenv.fish\n"
+      s << "  cp #{libexec}/fish/export.fish ~/.config/fish/functions/export.fish\n\n"
```

Comment:
Why `cp`, and not create symlinks? Copying requires updating every time `jenv.fish` and `export.fish` changes, and verifying whether it has can become quite involved.

Will the right place to create these copies/symlinks still be `~/.config/fish` if I've set `XDG_CONFIG_HOME` to something else?

Reply:
thanks, updated

> Will the right place to create these copies/symlinks still be ~/.config/fish if I've set XDG_CONFIG_HOME to something else?

I'm not sure. What's the best way here?